docker-compose.yml

```
@@ -29,3 +29,4 @@ services:
 
 volumes:
   mysql-data:
+  redis-data:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)